utils/sl/components/__init__.py

```diff
@@ -217,6 +217,9 @@ def define_common(reqs):
     reqs.define('crypto', libs=['crypto'], headers=['openssl/md5.h'],
                 package='openssl-devel')
 
+    reqs.define('json-c', libs=['json-c'], headers=['json-c/json.h'],
```

Comment:
is this built on top of Hua's patch ?

Reply:
Yep. I needed to get the dmg-based pool management stuff working for my prototype that moves the management service up into the control plane, so I just went ahead and did it as part of that task.